HISTORY.md

```diff
@@ -1,3 +1,10 @@
+2019-11-02 / 2.0.0
+==================
+
+  * deps: depd@2.0.0
+     - Drop support for Node.js 0.6 + 0.8
```

Comment:
I think the 0.8 part is in the wrong area. Afaik depd 2 still supports 0.8, so if the desire is to drop 0.8 support, it should probably be listed at the level of this module's change (with the asterisk).

Reply:
But the test failed for 0.8 too

Reply:
That's strange, the `depd` has `>= 0.8` in the package.json (https://github.com/dougwilson/nodejs-depd/blob/master/package.json#L36) and also has a successful buld on Node.js 0.8 (https://travis-ci.org/dougwilson/nodejs-depd/jobs/602309636). What was the 0.8 failure? Maybe need to report up to the `depd` module the issue and look at resolving it.

The history entry for `depd` 2 only says 0.6 is dropped (https://github.com/dougwilson/nodejs-depd/blob/master/History.md#200--2018-10-26) not 0.8 like is written here.

Reply:
i reenabled 0.8 on travis and it works - sorry for the puzzle